DS13/code/game/objects/replicator.dm

```diff
@@ -190,14 +199,16 @@
 /obj/machinery/replicator/proc/replicate(var/what, var/temp, var/mob/living/user)
```

Comment:
`var/what` - Makes sense in this context, but is a bit confusing down the line. Perhaps rename to indicate it's the item that the replicator's replicating.

`var/temp` - If you know the BL, you know it means _temperature_. `temp` however is commonly used in programming to mean _temporary_, which makes it slightly confusing at first. Why not spell out the whole word?